data/countries_names.txt

```diff
@@ -52682,49 +52682,82 @@
   uk = Ісфахан, Шираз, Ахваз
 
 [Iraq_North Description]
-  en = Mosul, Erbil, Kirkuk
-  ar = اربيل, كركوك
-  az = Mosul, Ərbil
-  da = Kirkuk
-  de = Mossul, Arbil, Kirkuk
-  el = Μοσούλη, Ερμπίλ, Κιρκούκ
-  fr = Mossoul, Erbil, Kirkouk
-  he = כירכוכ
-  ja = モースル, アルビール, キルクーク
-  pl = Mosul, Irbil, Kirkuk
-  ru = Мосул, Эрбиль, Киркук
-  sv = Kirkuk
-  tr = Musul, Erbil
-  uk = Мосул, Ербіль, Кіркук
+  en = Mosul, Erbil, Kirkuk, Tikrit
+  ar = الموصل، أربيل، كركوك، تكريت
+  az = Mosul, Ərbil, Kərkük, Tikrit
+  be = Масул, Эрбіл, Кірук, Цікрыт
+  cs = Mosul, Erbíl, Kirkúk, Tikrít
+  da = Mosul, Erbil, Kirkuk, Tikrit
+  de = Mossul, Erbil, Kirkuk, Tikrit
+  el = Μοσούλη, Ερμπίλ, Κιρκούκ, Τικρίτ
+  es = Mosul, Erbil, Kirkuk, Tikrit
+  et = Mosul, Erbil, Kirkuk, Tikrit
+  fi = Mosul, Erbil, Kirkuk, Tikrit
+  fr = Mossoul, Erbil, Kirkouk, Tikrit
+  he = מוסול, ארביל, כירכוכ, תיכרית
+  hu = Moszul, Erbíl, Kirkuk, Tikrit
+  it = Mosul, Erbil, Kirkuk, Tikrit
+  ja = モスル, エルビル, キルクーク, ティクリート
+  ko = 모술, 에르빌, 키르쿠크, 티크리트
+  nl = Mosul, Erbil, Kirkuk, Tikrit
+  pl = Mosul, Irbil, Kirkuk, Tikrit
+  pt = Mossul, Erbil, Kirkuk, Tikrit
+  ro = Mosul, Erbil, Kirkuk, Tikrit
+  ru = Мосул, Эрбиль, Киркук, Тикрит
+  sk = Mosul, Erbíl, Kirkúk, Tikrít
+  sv = Mosul, Erbil, Kirkuk, Tikrit
+  th = โมซูล, เออร์บิล, เคอร์คุก, ติกริต
+  tr = Musul, Erbil, Kerkük, Tikrit
+  uk = Мосул, Ербіль, Кіркук, Тікріт
+  vi = Mosul, Erbil, Kirkuk, Tikrit
+  bg = Мосул, Ербил, Киркук, Тикрит
+  hr = Mosul, Erbil, Kirkuk, Tikrit
+  lt = Mosulas, Erbilas, Kirkukas, Tikritas
+  lv = Mosula, Erbīla, Kirkūka, Tikrita
+  no = Mosul, Erbil, Kirkuk, Tikrit
+  sr = Мосул, Ербил, Киркук, Тикрит
+  sl = Mosul, Erbil, Kirkuk, Tikrit
+  zh-hans = 摩苏尔, 埃尔比勒, 基尔库克, 提克里特
+  zh-hant = 摩蘇爾, 埃爾比勒, 基爾庫克, 提克里特
```

Comment:
1. It should be zh-Hans and zh-Hant (capital H)
2. Translations should be sorted, here is the validation check that fails if translations are not sorted: https://github.com/organicmaps/organicmaps/actions/runs/9798311426/job/27056526538

Reply:
My apologies. I just edit it and made the required changes to fix the errors.
zh-hans and zh-hant to zh-Hans and zh-Hant.     
And order the languages alphabetically, the  entries (bg, hr, lt, lv, no, sr, sl, zh-Hans, zh-Hant) were not in alphabetical order before.    
 https://github.com/organicmaps/organicmaps/actions/runs/9911301077/job/27383649042?pr=8626